DockerFiles/docker_file_kasm_ubuntu_jammy_DISCOWER

```diff
@@ -0,0 +1,161 @@
+FROM kasmweb/ubuntu-jammy-desktop:1.14.0-rolling
+USER root
+
+#RUN rm /bin/sh && ln -s /bin/bash /bin/sh
+#SHELL ["/bin/bash", "-c"]
+
+ENV HOME /home/kasm-default-profile
+ENV STARTUPDIR /dockerstartup
+ENV INST_SCRIPTS $STARTUPDIR/install
+WORKDIR $HOME
+
+######### Customize Container Here ###########
+
+
+
+ARG KUH=/home/kasm-user
+RUN apt-get update && apt-get upgrade -y && \
+    apt-get install -y libomp5 && \
+    apt-get install -y sudo zsh && \
+    rm -rf /var/lib/apt/list/* && \
+    echo 'kasm-user ALL=(ALL) NOPASSWD: ALL' >> /etc/sudoers && \
+    true
+    
+# Cleaning up junk
+#RUN apt-get purge -y thunderbird* && \
+#    apt-get remove -y google-chrome-stable gimp zoom signal-desktop obs-studio onlyoffice-desktopeditors sublime-text && \
+#    apt-get autoremove -y && \
+#    apt-get clean
+    
+#RUN rm -f /home/kasm-user/Desktop/com.obsproject.Studio.desktop && \
+#    rm -f /home/kasm-user/Desktop/gimp.desktop && \
+#    rm -f /home/kasm-user/Desktop/google-chrome.desktop && \
+#    rm -f /home/kasm-user/Desktop/nextcloud.desktop && \
+#    rm -f /home/kasm-user/Desktop/onlyoffice-desktopeditors.desktop && \
+#    rm -f /home/kasm-user/Desktop/signal-desktop.desktop && \
+#    rm -f /home/kasm-user/Desktop/sublime_text.desktop && \
+#    rm -f /home/kasm-user/Desktop/telegram.desktop && \
+#    rm -f /home/kasm-user/Desktop/Zoom.desktop && \
+#    rm -f /home/kasm-user/Desktop/thunderbird.desktop                        
```

Comment:
Is it worth using kasm if there is so much stuff that is not needed? We are pulling and then deleting stuff again.

Reply:
I agree with this @joris997 , is it possible to trim their original docker so that it doesn't install all this by default?

Reply:
I'm using kasm for the simple browser GUI support. In a command above, I mention that there is a clean jammy docker with kasm: `https://hub.docker.com/r/kasmweb/core-ubuntu-jammy`
but I haven't had time to test what will break w.r.t. the installation instructions.

I'm not that familiar with docker, so if there is a more lightweight option to get browser GUI support (or you want to go forward with another interface method), then let me know!

Reply:
The best approach would be start barebone and include only the necessary dependencies. This helps a) to identify dependencies so you can maintain them b) decrease the size of the docker, so you can save resources. This is not only on your local system, but has a huge impact when running CI.

If you are overwhelmed by this, PX4 already has a container maintained which includes a minimum set of dependencies: https://github.com/PX4/PX4-containers This may not be the minimum, but quite close.

Reply:
Does that provide a VNC instance ? Joris showed it running next to me and it seems convinient to just open the browser and have a desktop environment.

Reply:
If it doesn't have one we can add it :smile:

Reply:
that would indeed be a nicer and lightweight solution. I will look into this @Jaeyoung-Lim